src/tools/log-parser-rs/src/args.rs

```diff
@@ -0,0 +1,86 @@
+use clap::{Parser, ValueEnum};
+use std::path::PathBuf;
+
+#[derive(Parser, Debug)]
+#[command(name="kata-log-parser", author, version, about, long_about = None)] // Read from `Cargo.toml`
+pub struct Cli {
+    pub input_file: PathBuf,
+
+    #[arg(short, long)]
+    pub output_file: Option<PathBuf>,
+
+    #[arg(short, long, help = "check log files and only display output on error")]
+    pub check_only: bool,
+
+    //TODO NOTE:(Not sure if this is still required, or why it needs output file set. (is only here
+    //because it was an arg for the go version.)
+    #[arg(
+        short,
+        long,
+        help = "display debug information (requires '--output-file')",
+        requires = "output_file"
+    )]
+    pub debug: bool,
+
+    //TODO NOTE:currently equivilant to error-if-no-records. Will change with multifile support
+    #[arg(long, help = "error if any files are empty")]
+    pub error_if_file_empty: bool,
+
+    //TODO NOTE:the original documentation didnt mention the abillity to include multiple log
+    //files, I assume one could just include as many filnames at the end as you wanted?
+    #[arg(long, help = "error if all logfiles are empty")]
+    pub error_if_no_records: bool,
+
+    //TODO
+    #[arg(
+        long,
+        help = "do not make an error for lines with no pid, source, name, or level"
+    )]
+    pub ignore_missing_fields: bool,
+
+    //TODO NOTE: Will probably have to wait on embeded log entries being standardized. (I really
+    //need to open that issue)
+    #[arg(long, help = "do not unpack agent log entries")]
+    pub no_agent_unpack: bool,
+
+    #[arg(
+        short,
+        long,
+        help = "suppress warning messages (incompatible with debug)",
+        conflicts_with = "debug"
+    )]
+    pub quiet: bool,
+
+    #[arg(
+        short,
+        long,
+        help = "do not tolerate misformed agent messages (may be caused by non-Kata Containers log lines)"
+    )]
+    pub strict: bool,
+
+    //TODO NOTE: Definitly an after-merge feature. Should I remove (or comment-out) this arg (and
+    //the corresponding enum) entirely untill then?
+    #[arg(long, value_enum, default_value_t = InputFormat::KataRs, help="set the format of the input logs, old go based runtime vs new rust-based runtime.")]
+    pub input_format: InputFormat,
```

Comment:
Should I just remove this argument (or comment it out) for now?

Reply:
I'd remove it for now as we avoid commented out code to keep the codebase as clean as possible as a rule. Maybe cut'n'paste into a GH issue so we don't forget about it.